os/arch/xtensa/src/xtensa/xtensa_usestack.c

```diff
@@ -0,0 +1,161 @@
+/******************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+/****************************************************************************
+ * arch/xtensa/src/common/xtensa_usestack.c
```

Comment:
Plaese modify the file path according to the location.

Reply:
Yes, I didn't remove it becasue it is in nuttx licence. I will remove it and check other files. Thanks.